### PR TITLE
Feature/ratings,simplified grid

### PR DESCRIPTION
**Pull request by thegeorgenikhil**
added:
- grid
- sidebar on mobiles
- ratings

### CODE REVIEW

File: components/Rating/rating.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+const ratingStars = document.getElementsByClassName("rating-star");
+for (let i = 0;i < ratingStars.length;i++){
+    ratingStars[i].addEventListener("click",(e) => {
+        const id = ratingStars[i].id;
+        const idValue = id.slice(id.length-1);
+        console.log(idValue)
```
Comment (romabulani) [inline components/Rating/rating.js]: Just a suggestion You can remove the console log here

**Review by romabulani — APPROVED** [review]
Good to merge

File: components/Rating/rating.js (hunk 1 above)
Comment (thegeorgenikhil) [reply components/Rating/rating.js]: will fix that!

**Review by Shah-Arsalan — APPROVED** [review]
Looks fine to me

**Pull request merged**